### ISSUE TITLE
Failure to start due to esmoothchunkr 

### ISSUE THREAD

**Issue by leftovernick** [LunaPixelStudios/Better-MC#987]
### Describe the Bug

Going through the logs, the server is failing to start due to a versioning conflict. Forge is running on 1.19.2

```
net.minecraftforge.fml.ModLoadingException: Mod esmoothchunkr requires 6minecraftr o1.17 or above, and below 1.19r
7Currently, 6minecraftr7 is o1.19.2
```

I can remove this mod along with two others that throw errors and the server will start, but i've failed to connect client side.

### Reproduction Steps

1. Using Crafty Controller
2. Server starts fine before adding the mod pack
3. After pasting config and mods folders attempt to start server
4. Observer crash

### Crash Report or Log

https://paste-bin.xyz/973132

### Operating System

Docker

### Minecraft Version

1.19

### Modloader

Forge

### Modpack Version

1.19.2

### Optifine

No

**Comment by chorbintime** [comment]
How did you install the modpack, step by step?

**Comment by SHXRKIIE** [comment]
smoothchunkr requires minecraftr 1.17 or above, and below 1.19. Remove smooth chunk mod

**Comment by OtisWong** [comment]
> How did you install the modpack, step by step?

I have the same error:
i downloaded the mod pack
downgraded from forge 1.19.3 to 1.19.2
dropped the folder with all the mods into `C:\Users\username\AppData\Roaming\.minecraft/` and renamed the folder to mods
launched game as usual and got the error. if you try taking the chunk smoother out you just get more errors

**Comment by leftovernick** [comment]
@chorbintime sorry for the delay. I'm using Crafty Controller. So the steps would be as followed:
1. create new server
2. set server type to modded
3. set server to forge
4. set server version to 1.19.2
5. build server
6. go into server files and replace with files from mod pack
7. rebuild server to initiate all of the mods
8. observe the errors mentioned

**Comment by chorbintime** [comment]
You need to use the mods from the serverpack, which can be found on the modpack's Curseforge page.

**Comment by leftovernick** [comment]
@chorbintime Yes, I am. I've downloaded ServerPack 1.19.2 and moved the mods folder, config folder and all the contents over to the server folder. It is attempting to load said mods, but throwing the error on smoothchunk which is one of the included server pack mods.

`Mod smoothchunk requires minecraft 1.17 or above, and below 1.19
Currently, minecraft is 1.19.2)`
